readme.md

```diff
@@ -5,37 +5,50 @@
 [![GitHub](https://img.shields.io/github/license/ros-wg-delib/awesome-ros-deliberation?style=flat-square)](LICENSE) [![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/ros-wg-delib/awesome-ros-deliberation/awesome-lint.yml?style=flat-square)](https://github.com/ros-wg-delib/awesome-ros-deliberation/actions)
 > A curated list of awesome tools, libraries and resources for deliberation in ROS 2.
 
-Deliberation is the topmost layer in a robotics architecture sometimes also called mission or planning layer. It aims at controlling the robots behavior towards its extended goal or function. This includes pre-programmed state machines, automated symbolic planning as well as behavioral reaction to unforeseen situations happening at runtime.
+Deliberation is the topmost layer in a robotics architecture sometimes also called mission or planning layer.
+It aims at controlling the robots behavior towards its extended goal or function.
+This includes pre-programmed state machines, automated symbolic planning as well as behavioral reaction to unforeseen situations happening at runtime.
```

Comment:
```suggestion
Deliberation is the topmost layer in a robotics architecture sometimes also called mission or planning layer. It aims at controlling the robots behavior towards its extended goal or function. This includes pre-programmed state machines, automated symbolic planning as well as behavioral reaction to unforeseen situations happening at runtime.
```

Reply:
Hah, sorry, my brain defaults to this principle even in other people's repos.

https://nick.groenen.me/notes/one-sentence-per-line/

Reply:
Oh always assumed this would not work in markdown. I also do my latex like this.
Thanks for teaching me 😊 https://github.com/ros-wg-delib/awesome-ros-deliberation/pull/4